pythonforandroid/bootstrap.py

```diff
@@ -263,11 +264,10 @@ def strip_libraries(self, arch):
             info('Python was loaded from CrystaX, skipping strip')
             return
         env = arch.get_env()
-        strip = which('arm-linux-androideabi-strip', env['PATH'])
```

Comment:
Since `which` is not used anymore, would you mind removing the import to please the CI :pray: 
```
pythonforandroid/bootstrap.py:12:1: F401 'pythonforandroid.util.which' imported but unused
```